src/app/App.js

```diff
@@ -0,0 +1,173 @@
+    'dijit/_WidgetsInTemplateMixin',
+
+    'dojo/dom',
+    'dojo/dom-style',
+    'dojo/on',
+    'dojo/text!app/templates/App.html',
+    'dojo/_base/array',
+    'dojo/_base/declare',
+    'dojo/_base/lang',
+
+    'esri/dijit/Print',
+    'esri/geometry/Extent',
+
+    'ijit/widgets/layout/SideBarToggler',
+
+    'layer-selector'
+], function (
+    FindAddress,
+    MagicZoom,
+    BaseMap,
+
+    config,
+    CoordinateSectorCreator,
+
+    registry,
+    _TemplatedMixin,
+    _WidgetBase,
+    _WidgetsInTemplateMixin,
+
+    dom,
+    domStyle,
+    on,
+    template,
+    array,
+    declare,
+    lang,
+
+    Print,
+    Extent,
+
+    SideBarToggler,
+
+    BaseMapSelector
+) {
+    return declare([_WidgetBase, _TemplatedMixin, _WidgetsInTemplateMixin], {
+        // summary:
+        //      The main widget for the app
+
+        widgetsInTemplate: true,
+        templateString: template,
+        baseClass: 'app',
+
+        // childWidgets: Object[]
+        //      container for holding custom child widgets
+        childWidgets: null,
+
+        // map: agrc.widgets.map.Basemap
+        map: null,
+
+        constructor: function () {
+            // summary:
+            //      first function to fire after page loads
+            console.info('app.App::constructor', arguments);
+
+            config.app = this;
+            this.childWidgets = [];
+
+            this.inherited(arguments);
+        },
+        postCreate: function () {
+            // summary:
+            //      Fires when
+            console.log('app.App::postCreate', arguments);
+
+            // set version number
+            this.version.innerHTML = config.version;
+
+            this.initMap();
+
+            this.childWidgets.push(
+                new SideBarToggler({
+                    sidebar: this.sideBar,
+                    map: this.map,
+                    centerContainer: this.centerContainer
+                }, this.sidebarToggle),
+                new CoordinateSectorCreator({
+                    map: this.map
+                }, this.coordinateNode)
+            );
+
+            this.inherited(arguments);
+
+            this.setupConnections();
+        },
+        setupConnections: function () {
+            // summary:
+            //      Fires when
+            console.log('app.App::setupConnections', arguments);
+
+            on.once(this.egg, 'dblclick', lang.hitch(this, 'showLevel'));
+        },
+        startup: function () {
+            // summary:
+            //      Fires after postCreate when all of the child widgets are finished laying out.
+            console.log('app.App::startup', arguments);
+
+            var that = this;
+            array.forEach(this.childWidgets, function (widget) {
+                console.log(widget.declaredClass);
+                that.own(widget);
+                widget.startup();
+            });
+
+            this.inherited(arguments);
+        },
+        showLevel: function () {
```

Comment:
The text for the level gets covered by the "g" in the title:
![image](https://cloud.githubusercontent.com/assets/1326248/15324255/ef499f62-1c02-11e6-86aa-6982d83c4852.png)

I would suggest just removing the egg stuff from this project.